doc/api/deprecations.md

```diff
@@ -2439,6 +2439,19 @@ The `_channel` property of child process objects returned by `spawn()` and
 similar functions is not intended for public use. Use `ChildProcess.channel`
 instead.
 
+<a id="DEP0130"></a>
+### DEP0130: Module.createRequireFromPath()
```

Comment:
Should this be `DEP00XX` or something like that? https://github.com/nodejs/node/blob/be6b1a2b3a5e01491b829ac9a5ed7602807af2de/doc/releases.md#step-3-update-any-replaceme-and-dep00xx-tags-in-the-docs

Reply:
fixed